diff --git a/dist/module.mjs b/dist/module.mjs
index 1537af32b9a1283b15091b9951a96161def13d14..758296a5a72898a213f9723122dec00aaba092c5 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -14,6 +14,7 @@ const version = "0.5.16";
 
 function generateWrangler(hub) {
   return [
+    'account_id = "83430b3b7efdba7efceccf3a2f921042"',
     hub.analytics ? [
       "analytics_engine_datasets = [",
       '  { binding = "ANALYTICS", dataset = "default" }',
@@ -34,7 +35,12 @@ function generateWrangler(hub) {
       "d1_databases = [",
       '  { binding = "DB", database_name = "default", database_id = "default" }',
       "]"
-    ] : []
+    ] : [],
+    hub.ai ? [
+      "[ai]",
+      'binding = "AI"',
+      ""
+    ] : [],
   ].flat().join("\n");
 }
 function addDevtoolsCustomTabs(nuxt, hub) {
